test/unit/get-config-test.js

```diff
@@ -240,4 +241,18 @@ describe('get-config', function() {
     expect(secondMessage).to.not.be.ok;
   });
 
+  it('does no mutate the config', function() {
```

Comment:
@kellyselden - This test looks good, can you confirm it failed prior to the other changes?

Reply:
confirmed it failed terribly on master!

Reply:
awesome, thank you!